src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -114,11 +108,8 @@ public function createBuilder()
 
     /**
      * Add a RouteCollectionBuilder.
-     *
-     * @param string                 $prefix
-     * @param RouteCollectionBuilder $builder
      */
-    public function mount($prefix, self $builder)
+    public function mount(string $prefix, self $builder)
```

Comment:
prefix is a string here but could also be null in https://github.com/symfony/symfony/blob/d321a1ab5dc9ba3a9ccec2adf80ef7d3c6d2ca6c/src/Symfony/Component/Routing/RouteCollectionBuilder.php#L57 which would currently break.
but the prefix handling in this class seems strange anyway. `mount` will make the prefix an empty string when prefix is `/` (the default). but in https://github.com/symfony/symfony/blob/d321a1ab5dc9ba3a9ccec2adf80ef7d3c6d2ca6c/src/Symfony/Component/Routing/RouteCollectionBuilder.php#L262 it checks for null. IMO the prefix should be initalized as string and then checked for `''` instead.

but I would ask you to not add types in RouteCollectionBuilder because I proposed to deprecate it in https://github.com/symfony/symfony/issues/32240